content/common/contributing-docs.md

```diff
@@ -197,6 +197,34 @@ test.sh
 {{ blockEnd }}
 ```
 
+**Note**: Existing documentation will already be using class names for code block `tabs` and `startTab` i.e. `{{ tabs "os" }}` and `{{ startTab "Windows"}}` respectively. Please consult the following `tabs` and `startTab` class names that are already in use (before creating your own). If you need to create a new class name (because one does not already exist) please add it to the list below as part of the pull request that contains your code block contribution.
+
+**tabs**:
+- `gh-interact`
+- `os`
+- `sdk-type`
+- `spin-version`
+
+**startTab**
```

Comment:
Is it worth mentioning under which `tabs` block these are typically used?

Reply:
Thinking hard about this one ...
I can't come up with a reason to associate the relationship between `tabs` and `startTab`. The main thing that leans toward not listing associations is that there may be cases where generic `startTab` names might fall under more than one `tabs` parent. Versions spring to mind initially. I tried using a table but then thought this list format was easier to scan (alphanumeric ordering) and also easier for a contributor to edit.

Reply:
TL;DR I am happy with just listing these separately.

Reply:
An example of why might be I am creating an `os` block, now what tabs should I have - Linux and Windows, okay, but macOS, Mac OS, Mac OS X, OS X, or Darwin?  For SDK type is it Go or TinyGo, C Sharp or C#?  Some kind of indicative "likely tabs" would help authors see this more readily.

Though realistically I always just copy tab blocks from somewhere else anyway _scurries away_

Reply:
Same :)